src/main.js

```diff
@@ -1,7 +1,12 @@
+/**
+ * @fileoverview Configure your project for ninja's parallel and incremental
+ *               builds with pure js.
+ */
+
 let ArgumentParser = require('argparse').ArgumentParser;
 let FindStream = require('./find_stream');
+let fs = require('fs');
```

Comment:
I think nothing in this file needs to change. Let's try to keep the changeset small.